extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -395,6 +396,7 @@ export class AmpStoryEmbeddedComponent {
     event.stopPropagation();
 
     this.setState_(ComponentState.EXPANDED, this.previousTarget_);
+    this.animateExpanded_(devAssert(this.previousTarget_));
```

Comment:
Shouldn't this line be in the `toggleExpandedView_` (or `setState`?) method, so we have all the UI updates triggered from the same place?

Reply:
Done.